Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -562,7 +562,11 @@ void VolumeVerifier::CheckMisc()
   }
   else
   {
-    const char country_code = game_id_encrypted[3];
+    char country_code;
+    if (IsDisc(m_volume.GetVolumeType()))
+      country_code = game_id_encrypted[3];
+    else
+      country_code = static_cast<char>(m_volume.GetTitleID().value_or(0) & 0xff);
```

Comment:
I wonder if we should add a `Volume::GetCountryCode` method or something along those lines? `game_id_encrypted[3]` was cryptic enough for the onlooker, and using the low byte of the title id in some other case isn't really that much nicer.

Also, isn't `GetTitleID` sane for discs, so we can use the `else` in all cases?

Reply:
> I wonder if we should add a `Volume::GetCountryCode` method or something along those lines? `game_id_encrypted[3]` was cryptic enough for the onlooker, and using the low byte of the title id in some other case isn't really that much nicer.

Maybe, but we would only really be using it in this one place, so it feels a bit weird...

> Also, isn't `GetTitleID` sane for discs, so we can use the `else` in all cases?

Only Wii titles have title IDs, so we can't do that for GameCube discs.

Reply:
Ugh, Title ID, Game ID, not the same thing :/
Nevermind that one then...

And even if its only used in one place, its not like they're the only ones (re. `GetContent`/`GetContentOffsets` from #8243).
The argument for doing it would be proper encapsulation, the argument against it is what you mentioned; just one place that uses it (and my threshold is usually above 2; so I'm ok if you want to leave it as-is when chances for reuse elsewhere are slim to none - since you're more likely to know if you'll need it in future PRs)